pages/mydoc/using_the_mobile_app.md

```diff
@@ -73,23 +73,23 @@ You can add as many rules as you see fit.
 
 ### Teams
 
-Team lists all the members of the organization. additionally, if the user is on call, there will be a small green dot against the user avatar. 
+Team lists all the members of the organization. Additionally, if the user is on call, there will be a small green dot against the user avatar. 
 
 ![](images/using_mobile_9.png){: style="max-width: 30%" }
 
-Navigating to the user will be taken into the user profile. User profile lists all the user info including the user notification rules, schedules escalation-policies and squads
+Navigating to the user will be taken into the user profile. User profile lists all the user info including the user notification rules, schedules escalation-policies and squads.
 
 ![](images/using_mobile_10.png){: style="max-width: 30%" }
 
 ### Schedules
 
-Schedules list's all the schedules organized against date and time, split into time and schedule
+Schedules list’s all the schedules organized against date and time.
```

Comment:
Schedules lists all the schedules organized against date and time.

Reply:
updated